.travis.yml

```diff
@@ -1,10 +1,24 @@
 language: rust
-rust:
-  - stable
-  - nightly
+
+ branches:
+   only:
+   - master
```

Comment:
this was supposed to avoid running tests twice (for a pr and for a branch):
![travis-twice](https://user-images.githubusercontent.com/4211399/47864313-53b1ca80-de0a-11e8-8ee7-ffdfc22e970c.png)
it works [for jsonrpc](https://github.com/paritytech/jsonrpc/blob/2ed142d55662155b0dc76b9b5d2732300f265dd6/.travis.yml#L3-L6), not sure why it doesn't here